src/theme/shared/shadows.ts

```diff
@@ -4,6 +4,7 @@ const shadows = {
   sm: '0px 4px 12px rgba(0, 0, 0, 0.05), 0px 8px 24px rgba(0, 0, 0, 0.1);',
   md: '0px 2px 6px rgba(0, 0, 0, 0.05), 0px 8px 24px rgba(0, 0, 0, 0.1), 0px 20px 48px rgba(0, 0, 0, 0.15)',
   lg: '0px 2px 8px rgba(0, 0, 0, 0.04), 0px 4px 16px rgba(0, 0, 0, 0.07), 0px 12px 32px rgba(0, 0, 0, 0.15), 0px 24px 64px rgba(0, 0, 0, 0.2)',
+  button: '0px 2px 0px #C2BF00',
```

Comment:
This might be tricky:
- We are hardcoding an off-brand colour here
- I would assume it needs to be a different colour for the ms button

Reply:
![image](https://user-images.githubusercontent.com/3371760/171818854-4245e1ae-284d-4af7-a69b-80ff56397038.png)
brand-300 is looking pretty close and will not give a yellow-green isch border on the ms version

Reply:
As a side note: the button shadow is logically already breaking the theme by adding a specific shadow for a single component